ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/host1/port-channel-interfaces.yml

```diff
@@ -946,3 +946,13 @@ port_channel_interfaces:
   - name: Port-Channel132
     description: Test_port-channel_interface-profile
     profile: test-interface-profile
+
+  - name: Port-Channel133
```

Comment:
let's add a test for traffic_engineering.enabled: false for port-channels as well (for coverage purpose)